src/Core/src/Platform/Android/NativeVersion.cs

```diff
@@ -1,27 +1,15 @@
+using System;
 using Android.OS;
 
 namespace Microsoft.Maui.Platform
 {
 	public static partial class NativeVersion
 	{
-		static readonly BuildVersionCodes BuildVersion = Build.VERSION.SdkInt;
+		public static bool IsAtLeast(BuildVersionCodes buildVersionCode) => OperatingSystem.IsAndroidVersionAtLeast((int)buildVersionCode);
 
-		public static bool IsAtLeast(BuildVersionCodes buildVersionCode)
-		{
-			return buildVersionCode >= BuildVersion;
-		}
+		public static bool IsAtLeast(int apiLevel) => OperatingSystem.IsAndroidVersionAtLeast(apiLevel);
```

Comment:
`NativeVersion.IsAtLeast()` and `Supports()` are public, is it useful at all? Should we delete these methods?

Reply:
Discussed on a call, I'll mark these `[Obsolete]`.

Reply:
The point of these was to have a cross-platform consistent way for us to do version checking and capabilities. That's why this is a partial class - so we can use these in handlers without having to use #ifdefs.

The intent was to wrap the OperatingSystem calls in the NativeVersion class: https://github.com/dotnet/maui/issues/461

Since we've shifted all the of the actual code in the handlers to platform-specific extension methods, maybe these aren't that useful? If that's the case, we don't need to obsolete them - we can just delete them. They're new with MAUI, and haven't seen a stable public release yet.

Reply:
The iOS one seems more useful, as it has this `string capability` overload:

https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/iOS/NativeVersion.cs

This PR is already getting huge -- so I think I'm going to leave these APIs here and just have them call `OperatingSystem` on Android.